script.js

```diff
@@ -1 +1,123 @@
+const form = document.querySelector(".input-box"),
+    input=form.querySelector("input"),
+    waitList=document.querySelector(".wait-list"),
+    completeList=document.querySelector(".complete-list");
 
+const TODO_LS='toDos';
+
+
+
+function loadToDos(){
+    const loadedToDos=localStorage.getItem(TODO_LS);
+    if(loadedToDos!==null){
+        const parsedToDos=JSON.parse(loadedToDos);
+        parsedToDos.forEach(function(todo){
+            paintToDo(todo.text);
+        });
+    }
+}
+
+function saveToDos(){
+    localStorage.setItem(TODO_LS,JSON.stringify(toDos));
+}
+const toDos=[];
+function paintToDo(text){
+    const plusLi = document.createElement('li');
+    const delBtn = document.createElement('img');
+
+    //const img=document.createElement("img");
+    delBtn.src="img/bin.png";
+    //delBtn.innerHTML="$";
+
+    //delBtn.append(img);
+    delBtn.addEventListener("click", deleteToDo);
+    const newId=toDos.length+1;
```

Comment:
스터디때 말씀드린 부분인데, todos.length에 의존하게 되면 삭제했을 때 id가 동일한 TODO가 여러 개 생길 수 있을 것 같아요. 이렇게 해보는 건 어떨까요?

```suggestion
    const newId= Date.now();
```

이 코드가 실행되는 유닉스 시간이 찍혀서 절대 겹칠 일이 없을 것 같아요~!